app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy(command: str) -> None:
+    command = command.split()
```

Comment:
You can split using a bunch of variables for better readability, f.e.
`cmd, file_name, new_file_name = command.split()`